cmd/snap-confine/ns-support.c

```diff
@@ -160,6 +164,18 @@ void sc_initialize_mount_ns(void)
 			die("cannot change propagation type to MS_PRIVATE in %s", sc_ns_dir);
 		}
 	}
+
+	if (!experimental_features) {
```

Comment:
I'd remove this check, it's not adding anything to the mechanics IMO.

Reply:
dropped